packages/dds/tree/src/simple-tree/api/snapshotCompatibilityChecker.ts

```diff
@@ -0,0 +1,67 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import type { JsonCompatibleReadOnly } from "../../util/index.js";
+import { toStoredSchema } from "../toStoredSchema.js";
+import { TreeViewConfigurationAlpha, TreeViewConfiguration } from "./configuration.js";
+import { SchemaCompatibilityTester } from "./schemaCompatibilityTester.js";
+import { generateSchemaFromSimpleSchema } from "./schemaFromSimple.js";
+import { decodeSimpleSchema, encodeSimpleSchema } from "./simpleSchemaCodec.js";
+import type { SchemaCompatibilityStatus } from "./tree.js";
+import { toSimpleTreeSchema } from "./viewSchemaToSimpleSchema.js";
+
+// TODO: Something more type-safe
+/**
+ * The JSON representation of a tree schema used for snapshot compatibility checking.
+ *
+ * @alpha
+ */
+export type JsonViewSchema = JsonCompatibleReadOnly;
+
+/**
+ * Checks the compatibility of the view schema which created the document against the view schema being used to open it.
+ *
+ * @param viewWhichCreatedDocument - The view schema which was used to create the document.
+ * @param view - The view schema being used to read the document.
+ * @returns - The compatibility status.
```

Comment:
`@returns` tags should not be followed by a `-`.
```suggestion
 * @returns The compatibility status.
```